<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LedControl</class>
 <widget class="QWidget" name="LedControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>309</width>
    <height>107</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LedControl</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Select LED to be switched on/off</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="led_4">
       <property name="text">
        <string>LED 4</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="led_3">
       <property name="text">
        <string>LED 3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="led_2">
       <property name="text">
        <string>LED 2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="led_1">
       <property name="text">
        <string>LED 1</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
